### DISCUSSION TITLE
Budget and Work Group Suggestions for Q1 2025

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#201] [General]
# Budget and Work Group Suggestions for Q1 2025

During today’s Steering Committee meeting(3 out 5 attended), we discussed RingDAO's proposal application process and the review of DCDAO's new member onboarding. Additionally, the meeting propose the following adjustments, considering that DCDAO should remain community-focused and aim to simplify management and governance complexity:

## 1. Marketing Management and Ecosystem Support (Furqan from Dotmatrix)
Tasks under Furqan’s Dotmatrix work group, including market operations, ecosystem support, website development, etc., can transition from DCDAO’s quarterly proposal format to a service provider model under the [**RingDAO Ecosystem Collaboration**](https://github.com/ringecosystem/collaboration/tree/master/service-providers). The budget remains unchanged at $3,500/month, totaling $10,500 per quarter, with payments switched to direct USDC transfers. Deliverables and reports will be submitted monthly, along with payments.  
(Recommendation: submit this service provider application by the end of December @nblogist . Example: [Pull Request #94](https://github.com/ringecosystem/collaboration/pull/94))

## 2. DCDAO's Community Growth Program Consolidation
Due to issues with PartnershipDude’s GitHub usage issue and reporting delays, it is recommended to merge the **Community Growth Program** into the **Community Manager Program**, to be managed and applied for by JDubs. The specific budget allocation and arrangements can remain the same or be adjusted upon @2xJDubs’ review, with applications submitted quarterly and reports submitted monthly.

**Comment by 2xJDubs** [comment]
This setup works for me.  I will work with Partnership Dude to incorporate his monthly reports into what I post and we will discuss budgets together working with the agreed upon framework set by the Quarterly Budget proposals.
